index.bs

```diff
@@ -4894,7 +4894,7 @@ TPM [=attestation certificate=] MUST have the following fields/extensions:
 - The Subject Alternative Name extension MUST be set as defined in [[!TPMv2-EK-Profile]] section 3.2.9.
 
 - The Extended Key Usage extension MUST contain the
-    "joint-iso-itu-t(2) internationalorganizations(23) 133 tcg-kp(8) tcg-kp-AIKCertificate(3)" OID.
+    "joint-iso-itu-t(2) internationalorganizations(23) 133 tcg-kp(8) tcg-kp-AIKCertificate(3)" (`2.23.133.8.3`) OID.
```

Comment:
Nit: in all the examples above the numerical numerical dot notation comes first, the human-readable notation (if available) second. It seems best to use that here as well. The numerical notation is probably what you'll want as an implementer and one can always figure out (parts of) the human-readable notation with some internet searching.

Reply:
Fixed in #1513, thanks! :smile: